### DISCUSSION TITLE
The UI web page block when select endpoints

### DISCUSSION THREAD

**Discussion by zhaoxiaojie0415** [apache/skywalking#10068] [General]
The error message is below.
<img width="1149" alt="image" src="https://user-images.githubusercontent.com/13363629/204995372-a0d49f36-470f-43e0-b62e-965fdff3dc90.png">

I am not sure whether it need to add unique index in the below code blocks or not.
<img width="1100" alt="image" src="https://user-images.githubusercontent.com/13363629/204995735-a574e353-3885-4787-ae7d-80e36f0d01b1.png">

**Comment by wu-sheng** [comment]
**SkyWalking v8 OAP server End-of-life(EOL)**

Ref, https://skywalking.apache.org/events/v8-eol/

We would not follow this report anymore.

**Reply by zhaoxiaojie0415** [reply]
ok, i got it. Thank you.
